daisy_workflows/image_import/debian/translate_debian_9.wf.json

```diff
@@ -5,6 +5,10 @@
       "Required": true,
       "Description": "The Debian 9 GCE disk to translate."
     },
+    "sysprep": {
+      "Value": "false",
+      "Description": "If enabled, run Windows Sysprep. Only applicable to Windows instances."
```

Comment:
Minor: consider changing description not to mention Windows but just state it's a sysprep flag that is currently no-op. Otherwise it might be confusing what Windows instances are being referred to here in the context of Linux.

Same for other Linux translate workflow files.